scripts/inference-dev

```diff
@@ -28,13 +28,17 @@ distDir="$cfiDir"/dist
 
 jdkPaths="${cfDir}"/jdk/annotated
 
-CFBuildDirs="${langtoolsDir}"/build/classes:"${annoToolsDir}"/scene-lib/bin:"${cfDir}"/dataflow/build:"${cfDir}"/javacutil/build:"${cfDir}"/stubparser/build:"${cfDir}"/framework/build:"${cfDir}"/checker/build
-CFIBuildDirs="${cfiDir}/bin"
+javacBuild="${langtoolsDir}"/build/classes
+
+CFBuild="${annoToolsDir}"/scene-lib/bin:"${cfDir}"/dataflow/build:"${cfDir}"/javacutil/build:"${cfDir}"/stubparser/build:"${cfDir}"/framework/build:"${cfDir}"/checker/build
```

Comment:
stubparser doesn't exist anymore in upstream. Please adapt to new location `stubparser/stubparser.jar` a level higher.

Reply:
Hi Prof. Dietl, where is the new location for `stubparser` dependency? It seems `stubparser` totally disappears in CF.

Reply:
Is the new location would be `framework/lib/stubparser-core.jar`?

Reply:
No, the stubparser is a separate project now. The standard build puts it here https://github.com/typetools/stubparser/blob/master/.travis-build-without-test.sh#L17
The integration into CF is ongoing: https://github.com/typetools/checker-framework/pull/1476
As stubparser is already merged, I would use that new location.